sdk/loadtestservice/Azure.Developer.LoadTesting/src/Azure.Developer.LoadTesting.csproj

```diff
@@ -1,10 +1,8 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>This is the LoadTestService client library for developing .NET applications with rich experience.</Description>
     <AssemblyTitle>Azure SDK Code Generation LoadTestService for Azure Data Plane</AssemblyTitle>
-    <Version>1.1.0-beta.1</Version>
-    <!--The ApiCompatVersion is managed automatically and should not generally be modified manually.-->
-    <ApiCompatVersion>1.0.2</ApiCompatVersion>
```

Comment:
we should not remove this line - if API Compat is reporting something, we should resolve them by adding customization code, or bump the major version to remove this.

Reply:
My bad, this was removed while development since it was breaking locally. Added it back